### PR TITLE
Dumb computer player

### PR DESCRIPTION
**Pull request by whaley-ethan**
Not sure how to test the contents of a private member variable of an object I created after I create the object (i.e. how do i test the result of lines 19-21 of Menu.java)

### CODE REVIEW

File: src/main/java/tic/tac/toe/Menu.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        this.inputOutput = inputOutput;
+    }
+
+    Game makeGame(Messages messages, InputOutput inputOutput, Board board, Player player1, Player player2) {
+        return new Game(messages, inputOutput, board, player1, player2);
+    }
+
+    public Game createGame() {
+        inputOutput.sendOutput(messages.greeting());
+        int numberOfHumanPlayers = 0;
```
Comment (ericmeyer) [inline src/main/java/tic/tac/toe/Menu.java:35]:
```suggestion
        int numberOfHumanPlayers = getNumberOfHumanPlayers()
```

File: src/main/java/tic/tac/toe/Menu.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        while (numberOfHumanPlayers > 2 || numberOfHumanPlayers < 1) {
+            inputOutput.sendOutput(messages.promptForPlayers());
+            numberOfHumanPlayers = inputOutput.getInput();
+        }
+
+        Player player1 = new HumanPlayer('X', inputOutput);
```
Comment (ericmeyer) [inline src/main/java/tic/tac/toe/Menu.java]:
```suggestion
        Player player1 = createPlayerOne(...);
```

File: src/main/java/tic/tac/toe/Menu.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        Player player2 = (numberOfHumanPlayers == 2) ?
```
Comment (ericmeyer) [inline src/main/java/tic/tac/toe/Menu.java]:
```suggestion
        Player player2 = createPlayerTwo(...);
```

**Review by ericmeyer — CHANGES_REQUESTED** [review]

File: src/main/java/tic/tac/toe/Menu.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                new HumanPlayer('O', inputOutput) :
+                new DumbComputer('O');
+        Board board = new Board();
+        Messages messages = new Messages();
+        Game game = makeGame(messages, inputOutput, board, player1, player2);
```
Comment (ericmeyer) [inline src/main/java/tic/tac/toe/Menu.java]:
```suggestion
        return makeGame(messages, inputOutput, board, player1, player2);
```

File: src/main/java/tic/tac/toe/Messages.java [hunk 5]
```diff
@@ -24,11 +24,14 @@ public String prompt(char currentPlayerSymbol) {
...
-    if (winningSymbol == TIE_GAME)
-    {
+    if (winningSymbol == TIE_GAME) {
       return "There is no winner! The only winning move is not to play!";
     } else {
       return "Congratulations to " + winningSymbol + ", our winner!";
     }
   }
+
+  public String promptForPlayers() {
```
Comment (whaley-ethan) [inline src/main/java/tic/tac/toe/Messages.java]: promptForNumberOfPlayers

**Pull request merged**